lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -165,6 +183,22 @@ def restart_evmserverd
         evmserverd_service = LinuxAdmin::Service.new("evmserverd")
         evmserverd_service.restart if evmserverd_service.running?
       end
+
+      def deactivate
+        @secure = nil
+        @server_host_is_ipaddr = nil
+        configure_messaging_type("miq_queue") # Settings.prototype.messaging_type = 'miq_queue'
+        restart_evmserverd
+        remove_installed_files
+      end
+
+      def secure?
+        @secure ||= server_port == 9_093
```

Comment:
`||=` won't cache as expected here because you can't cache a boolean properly with `||=`  (because false will always try again).

If we do want to cache this, this should be
```suggestion
        return @secure if defined?(@secure)

        @secure = server_port == 9_093
```

Reply:
That being, said, I wouldn't cache this at all, unless it's expensive, and an `==` is not expensive. (Not sure if server_port is a regular accessor or something more complicated)

Reply:
Thank you @Fryguy!
Resolved by commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/149/commits/14c07ec105fc32ca6fdf36842456fcde4a54d0a0